pennylane/devices/__init__.py

```diff
@@ -12,12 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-This subpackage provides reference plugin implementations. The reference plugins provide basic built-in qubit
+This subpackage provides default devices for PennyLane, which do not need external plugins to be installed.
```

Comment:
I think the idea of default devices changed over time :)

Reply:
I would have named it something catchier than `default.qubit` in retrospect if it wasn't just meant to be a reference plugin 😆